pages/patient/my_account/paymentMethodPage.js

```diff
@@ -0,0 +1,233 @@
+const elements = {
+    //menu items
+    personalInfoSection: `[data-test-id='personalInfoSection']`,
+    insuranceSection: `[data-test-id='insuranceSection']`,
+    allergiesSection: `[data-test-id='allergiesSection']`,
+    medicationsSection: `[data-test-id='medicationsSection']`,
+    conditionsSection: `[data-test-id='conditionsSection']`,
+    proceduresSection: `[data-test-id='proceduresSection']`,
+    familyHistorySection: `[data-test-id='familyHistorySection']`,
+    questionnaireSection: `[data-test-id='questionnaireSection']`,
+    pharmacySection: `[data-test-id='pharmacySection']`,
+    paymentsSection: `[data-test-id='paymentsSection']`,
+    settingsSection: `[data-test-id='settingsSection']`,
+
+    //request a visit button
+    btnRequestVisit: `data-test-id=['requestVisit']`,
+
+    //saved credit card
+    textFieldCaption: `.eVisitAppTextFieldCaption`,
+    savedCCNumber: `[name='number']`,
+    btnUpdateBilling: `[data-test-id='updateBillingInformation']`,
+
+    //MODALS:
+    //modal to erase saved credit card
+    modal: `.eVisitAppModal`,
+    btnConfirmModal: `[data-test-id="confirmModalConfirm"]`,
+    btnDenyModal: `[data-test-id="confirmModalDeny"]`,
+    //modal with CVC help
+    btnCloseHelpModal: `[data-test-id="cvcLocationModalClose"]`,
+
+    //add new card screen elements
+    ccName: `[data-test-id="cardHolderName"]`,
+    ccNumber: `[data-test-id="ccNumber"]`,
+    ccDate: `[data-test-id="expiryDate"]`,
+    ccCvc: `[data-test-id="cvcField"]`,
+    ccCvcHelp: `[data-test-id="cvcWhere"]`,
+    saveNewCreditCard: `[data-test-id="saveCreditCard"]`,
+
+    //alert banner on top if prohibited action happens. Ex: "There is more than one field that is required."
+    toast: `[data-test-id='toast']`,
+    //button to close alert banner
+    btnCloseToast: `[data-test-id='buttonCloseToast']`,
+
+};
+
+const commands = [{
+
+    /*
+    *    This function make sure the page is completelly loaded before continuing, using any specified element as a trait
+    */
+    accessPaymentMethodPage(email, password) {
+        this
+        const loginPage = this.api.page.loginPage()
+        const geolocationPage = this.api.page.patient.geolocationPage()
+        const landingPage = this.api.page.patient.landingPage()
+        loginPage
+            .goToPracticeLoginPage()
+            .patientLogin(email, password)
+        geolocationPage.confirmGeolocation()
+        landingPage.selectMyAccount();
+        this
+            .waitForElementVisible('@paymentsSection')
+            .pause(500)
+            .click('@paymentsSection')
+            .pause(1000)//This pause is necessary since there is no element to use as a trait in this page
+        return this
+    },
+
+    /*
+    *    This function will make sure there is no card registered
+    */
+    eraseRegisteredCard() {
+        let self = this;
+        this
+            .api.element(`@btnUpdateBilling`, function (result) {
```

Comment:
`self` isn't needed here if you just use an arrow function (this may not be possible depending on the JS engine(s) this is running on... just wanted to point it out).
```suggestion
            .api.element(`@btnUpdateBilling`, (result) => {
```

Reply:
Arrow functions ALWAYS bind to the scope they were created in, so the `this` of an arrow function always matches that of its parent scope.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
Thanks for the tip! It worked here as well. I will use this strategy from now on :)